src/app/test/db/mongo/util_test.py

```diff
@@ -365,19 +365,3 @@ def test_ensure_roles_creates_superuser_role_if_none_exists(self, monkeypatch):
         superuser = Role.objects.get(name="superuser")
 
         assert len(superuser.permissions) == len(Permissions)
-
-    def test_ensure_roles_preserves_existing_superuser(self, monkeypatch):
```

Comment:
The things this was testing for is not intended to be true with the changes to this PR.  Thus the removed test.